src/rprblender/export/light.py

```diff
@@ -146,6 +146,8 @@ def sync(rpr_context: RPRContext, obj: bpy.types.Object, instance_key=None):
     elif light.type == 'AREA':
         data = mesh.MeshData.init_from_shape_type(rpr.shape, light.size, light.size_y, segments=32)
         area = data.area * obj.scale[0] * obj.scale[1]
```

Comment:
Please provide one more fix here:
area = abs(data.area * obj.scale[0] * obj.scale[1])
because obj.scale could be below zero.